qemu/tests/fwcfg64.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from avocado.utils import process
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_disk
+from provider import win_dump_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Fwcfg64 basic function test:
+
+    1) Boot guest with -device vmcoreinfo.
+    2) Check the fwcfgg64 driver has been installed.
+    3) Run "dump-guest-memory -w memory.dmp" in qemu monitor.
+    4) Check the memory.dmp can be saved and the size is larger then 0Kb.
+    5) Check the dump file can be open with windb tools.
+    """
+    def set_vm_param():
```

Comment:
Will this function be used by other fwcfg64 test cases? If so, I suppose it also could be added to win_dump_utils.py.

Reply:
I'll move this function to win_dump_utils.py, thanks~